Gruntfile.coffee

```diff
@@ -54,6 +54,10 @@ module.exports = (grunt) ->
         src: 'dist/client.js'
         dest: 'dist/client.min.js'
 
+      instrumentation_jquery:
```

Comment:
Is it bundling it as default, removing the need to include it separately, like this? https://github.com/airbrake/airbrake-js/commit/174bc51dc71e5b2dc9864efb72f742f3553c5a6a